conan/profiles/inteldpcpp_lnx

```diff
@@ -30,8 +30,8 @@ compiler.libcxx=libstdc++11
 build_type=Release
 
 [build_requires]
-ninja/1.9.0
-cmake/[>=3.13]
+cmake/[>=3.15]
+ninja/1.10.0
```

Comment:
We need to update supported versions in README as well

Reply:
Done